zerver/tests/test_events.py

```diff
@@ -2354,6 +2355,44 @@ def test_do_deactivate_user(self) -> None:
         self.assert_on_error(error)
 
     def test_do_reactivate_user(self) -> None:
+        user_reactivate_checker = self.check_events_dict([
+            ('type', equals('realm_user')),
+            ('op', equals('add')),
+            ('person', check_dict_only([
+                ('user_id', check_int),
+                ('email', check_string),
+                ('avatar_url', check_none_or(check_string)),
+                ('full_name', check_string),
+                ('is_admin', check_bool),
+                ('is_bot', check_bool),
+                ('is_guest', check_bool),
+                ('is_active', check_bool),
+                ('profile_data', check_none_or(check_dict_only([]))),
+                ('timezone', check_string),
+                ('date_joined', check_string),
+            ])),
+            ('last_active_timestamp', check_none_or(check_int)),
+        ])
+
+        peer_add_event_checker = self.check_events_dict([
+            ('type', equals('subscription')),
+            ('op', equals('peer_add')),
+            ('subscriptions', check_list(check_string)),
+            ('user_id', check_int),
+        ])
+
+        user = self.example_user("cordelia")
+        do_deactivate_user(user)
+        update_user_presence(user, get_client("website"),
+                             timezone_now(), UserPresence.ACTIVE, False)
+        action = lambda: do_reactivate_user(user)
+        events = self.do_test(action, num_events=2)
+        error = user_reactivate_checker('events[0]', events[0])
```

Comment:
Maybe the name of this error variable should be more clear about the error it's expecting? The same for the other error variable below.

Reply:
In ideal situation it should not show error, but it has somehow be named error in all the tests written in this file and I have just followed the convention.